CHANGELOG.rst

```diff
@@ -12,9 +17,20 @@ Gitlab deployment as specified in our `documentation <https://renku.readthedocs.
 before installing this or any subsequent Renku version. Gitlab publishes an official Helm chart and
 that is what should be used for deploying Gitlab with Helm.
 
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**🌟 New Features**
+
+- **UI**: Configure and save session secrets in Renku 2.0 projects and use them in sessions (TODO).
```

Comment:
I think that is a leftover from editing?

```suggestion
- **UI**: Configure and save session secrets in Renku 2.0 projects and use them in sessions.
```

Reply:
Fixed in c023be3629f0d063da801d9f582fe3a8b9f39a47.